issue templates (#323)
nodejs/i18n: .github/ISSUE_TEMPLATE/language-request.md

---
name: Language request

> about: Request adding not existed language to Crowdin

Reviewer: ```suggestion
about: Request adding a new language to our Crowdin projects
```

> labels: crowdin,language request

Reviewer: Do these labels already exist on the repo?

Author: Yes, it is
